MinnPost/object-sync-for-salesforce
# 264 update gulp

## What does this PR do?

Convert's all of this plugin's Gulp tasks to their Gulp 4 equivalents. This fixes #264.

## How do I test this PR?

- `npm install`
- `gulp`

Make sure nothing bad happens with JavaScript or CSS files.
